components/d2l-quick-eval/build/lang/en.js

```
@@ -46,6 +46,7 @@ export const LangEn = {
 	'newPosts': 'New Posts',
 	'newSubmissionDetails': '{newNum, plural, =0 {{resub, plural, =1 {1 resubmission} other {{resub} resubmissions}}} other {{resub, plural, =0 {{newNum} new} =1{{newNum} new, 1 resubmission} other {{newNum} new, {resub} resubmissions}}}}',
 	'newSubmissions': 'New Submissions',
+	'nextSubmission': 'Next Submission',
```

this should trigger serge. But it will silently fail in other languages. Are we okay with this?

---

Since we will be testing other languages this iteration, I would prefer if master worked without any finagling. I think after you get a green review you should add it to all the language files.

---

the review is red because of the visual diff tests. If i get an Okay from you guys i'll update the goldens

---

Carol is still on vacation, maybe ask Dylan? Or probably 1 approval is enough for this particular change

---

Shouldn't the lang term be added to all of the .js files, not just en.js? Also add the lang term to en.json?

---

Goldens look fine to me... As long as there aren't any issues with the dialog resizing and the overflow shadow.

---

> Shouldn't the lang term be added to all of the .js files, not just en.js? Also add the lang term to en.json?

Yeah alex seems to have said the same thing.





> Updating goldens seems fine to me... As long as there aren't any issues with the dialog resizing and the overflow shadow.

Seems fine to me https://s3.ca-central-1.amazonaws.com/visualdiff.gaudi.d2l/screenshots/BrightspaceHypermediaComponents/activities/action-dismiss-dialog/2019-12-30-18.39.28.688-6160715578f00236ff1201eedf5380530df08022-report.html